app/parse.py

```diff
@@ -15,8 +30,102 @@ class Product:
+        self.driver.get(url)
+        cookies_accept_button = self.driver.find_element(
+            By.CLASS_NAME, "acceptCookies"
+        )
+        if cookies_accept_button.is_displayed():
+            cookies_accept_button.click()
+
+    def scroll_down_page_with_more_button(self, url: str) -> None:
+        self.driver.get(url)
+
+        while True:
+            try:
+                more_button = self.driver.find_element(
+                    By.CSS_SELECTOR, "[class*=scroll-more]"
+                )
+                if more_button.is_displayed():
+                    more_button.click()
+                    time.sleep(1)
+            finally:
+                break
+
+    def scrape_single_page(self, url: str) -> [Product]:
+        result = []
+        self.driver.get(url)
+        products = self.driver.find_elements(
+            By.CSS_SELECTOR, "[class*=product-wrapper]"
+        )
+        for product in products:
+            result.append(
+                Product(
+                    title=product.find_element(
+                        By.CSS_SELECTOR, "a.title"
+                    ).get_attribute("textContent"),
+                    description=product.find_element(
+                        By.CSS_SELECTOR, "[class*=description]"
+                    ).text,
+                    price=float(
+                        product.find_element(
+                            By.CSS_SELECTOR, "[class*=price]"
+                        ).text.replace("$", "")
+                    ),
+                    rating=len(
+                        product.find_elements(
+                            By.CSS_SELECTOR, "p > span.ws-icon-star"
+                        )
+                    ),
+                    num_of_reviews=int(
+                        product.find_element(
+                            By.CSS_SELECTOR, "[class*=review-count]"
+                        ).text.split()[0]
+                    ),
+                )
+            )
+        return result
+
+    @staticmethod
+    def write_to_csv(products: [Product], filename: str) -> None:
+        with open(
+                filename,
+                "w",
+                newline="",
+                encoding="utf-8"
+        ) as file:
+            writer = csv.writer(file)
+            writer.writerow(PRODUCT_FIELDS)
+            writer.writerows(
+                [asdict(product).values() for product in products]
+            )
+
+    def close_driver(self) -> None:
+        self.driver.quit()
+
+
 def get_all_products() -> None:
-    pass
+    for name, url in tqdm(
+        URLS.items(),
+        desc="Scrapping pages",
+        colour="green",
+    ):
+        webscraper = WebScraper()
+        webscraper.click_accept_cookies(url)
+        webscraper.scroll_down_page_with_more_button(url)
+        products = webscraper.scrape_single_page(url)
+        webscraper.write_to_csv(products, f"{name.lower()}.csv")
+        webscraper.driver.quit()
+        webscraper.close_driver()
```

Comment:
You scraper doesn't save url, thus you can move it out of this loop to not create it each time
OR if you want to use this way, then it makes sense to save url inside the scraper(as you loop through urls) and do not pass url in each method